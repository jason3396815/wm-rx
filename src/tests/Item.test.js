import { h, Component } from 'preact';
import Item from '../components/Item';

describe('<Item />', () => {
	it('should render a list item', () => {
		expect(
			<Item item={{
				title: 'Test Title',
				time_ago: '3 days ago',
			}} />
		).toMatchSnapshot();
	});
});

