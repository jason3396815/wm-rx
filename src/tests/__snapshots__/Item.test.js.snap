// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Item /> should render a list item 1`] = `
e {
  "attributes": Object {
    "item": Object {
      "time_ago": "3 days ago",
      "title": "Test Title",
    },
  },
  "children": Array [],
  "key": undefined,
  "nodeName": [Function],
}
`;
